tests/BuildScriptGeneratorCli.Tests/ExecCommandTest.cs

```diff
@@ -0,0 +1,57 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
+using McMaster.Extensions.CommandLineUtils;
+using Microsoft.Oryx.BuildScriptGenerator;
+using Microsoft.Oryx.Tests.Common;
+using Xunit;
+using Microsoft.Oryx.Common;
+using Microsoft.Oryx.BuildScriptGeneratorCli.Resources;
+
+namespace Microsoft.Oryx.BuildScriptGeneratorCli.Tests
+{
+    public class ExecCommandTest : IClassFixture<TestTempDirTestFixture>
+    {
+        private readonly TestTempDirTestFixture _testDir;
+
+        public ExecCommandTest(TestTempDirTestFixture testFixture)
+        {
+            _testDir = testFixture;
+        }
+
+        [Fact]
+        public void OnExecute_ExitsSuccessfully_WhenCommandIsEmpty()
+        {
+            // Arrange
+            var cmd = new ExecCommand { DebugMode = true, SourceDir = _testDir.RootDirPath };
+            var console = new TestConsole();
+
+            // Act
+            var exitCode = cmd.OnExecute(new CommandLineApplication(console), console);
+
+            // Assert
+            Assert.Equal(ProcessConstants.ExitSuccess, exitCode);
+        }
+
+        [Fact]
+        public void OnExecute_ShowsErrorAndExits_WhenNoUsableToolsAreDetected()
```

Comment:
Since our build image has LTS versions as default, IMO we should not throw this error and should run the provided command without erroring out. Unfortunately I do not know the usage of VS saas team here, but i will give a scenario: Let's say i start out with a directory which is empty and then provide a command like `dotnet new empty`...this should cd into the source directory and run this command and this will create a empty asp.net core project